test/jdk/jdk/jfr/jcmd/TestJcmdDump.java

```diff
@@ -105,8 +107,15 @@ private static void jfrDump(Boolean pathToGCRoots, String name, boolean disk, Pr
             leakList.add(new Object[1000_0000]);
             System.gc(); // This will shorten time for object to be emitted.
             File recording = new File("TestJCMdDump.jfr");
-            String[] params = buildParameters(pathToGCRoots, name, recording);
-            OutputAnalyzer output = JcmdHelper.jcmd(params);
```

Comment:
Not at all clear why you can't just add the additional timeout arg and still use the JcmdHelper ??

Reply:
I need to pass the the system property as the first argument. JcmdHelper uses the PidJcmdExecutor class in the test lib, which adds the PID as first argument, so I would need to rewrite both JcmdHelper and PidJcmdExecutor to support system properties functionality, which seemed overkill.